Packages/Sites/Neos.NeosIo/Resources/Private/Scss/Molecules/_BoxAside.scss

```diff
@@ -0,0 +1,29 @@
+.boxAside {
+	.main {
+
+		@media (max-width: 979px) and (min-width: 768px) {
```

Comment:
It's an exception for this case, as the media queries are simply not enough. Those should be optimized anyways but I won't have enough time for that now, which is why I took this approach for now.